TOPICS.md

```diff
@@ -0,0 +1,35 @@
+# Topics
+
+## Why Britecharts React
+Britecharts is a D3 based, high level charting library that allows users to create stunning data visualizations with little work. [React][react] is one of the most popular User Interface libraries that allows to create apps by composing small components in an intuitive way. Both libraries take control of the DOM, so how can we make them work together in a fluid way?
+
+Britecharts-react proposes a way of integrating Britecharts and React that promotes a **clear separation of concerns** between the libraries and aims for an **easy and effortless maintenance** of the charts.
```

Comment:
Nit: Britecharts-React

aims for **easy and effortless maintenance** of the charts.